MarvelApp/MarvelApp/Models/ComicModels/ComicBaseData.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+struct ComicBaseData: Codable {
+    let responseCode: Int
+    let apiDataSource: ComicBookDataSource?
```

Comment:
Use Generic type for this variable to reduce duplicate code!

ref: https://www.swiftbysundell.com/articles/creating-generic-networking-apis-in-swift/